python/tvm/topi/cuda/dense.py

```diff
@@ -44,6 +44,8 @@ def dense_cublas(cfg, data, weight, bias=None, out_dtype=None):
     matmul = cublas.matmul(data, weight, False, True)
     if isinstance(batch, int):
         cfg.add_flop(batch * in_dim * out_dim * 2)
+    else:
+        cfg.add_flop(batch.value * in_dim * out_dim * 2)
```

Comment:
Should have an assertion to make sure batch is a TVM container type.

Reply:
I've added a check that it is int or IntImm, would any other types show up here?

Reply:
I don't think so. Let's use the current version to make sure we could catch any exception.

Reply:
Ah it turns out that you may get Var in the case of dynamic shape. For other TOPI templates, many of them just ignore the flops when the shape is dynamic. You could do the same.